tfs/src/main/java/hudson/plugins/tfs/rm/ConnectReleaseWebHookEvent.java

```diff
@@ -0,0 +1,332 @@
+package hudson.plugins.tfs.rm;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import hudson.model.AbstractProject;
+import hudson.model.Descriptor;
+import hudson.model.Item;
+import hudson.plugins.tfs.TeamPluginGlobalConfig;
+import hudson.plugins.tfs.model.AbstractHookEvent;
+import hudson.plugins.tfs.model.servicehooks.Event;
+import hudson.tasks.Publisher;
+import hudson.util.DescribableList;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.InvalidParameterException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+import jenkins.model.Jenkins;
+import net.sf.json.JSONArray;
+import net.sf.json.JSONObject;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * This HookEvent is for to create/delete Release Webhook.
+*/
+public class ConnectReleaseWebHookEvent extends AbstractHookEvent {
+
+    private static final Logger logger = Logger.getLogger(ReleaseWebHookAction.class.getName());
+    private static final String CREATE_WEBHOOK_EVENTNAME = "rmwebhook-create";
+    private static final String REMOVE_WEBHOOK_ENENTNAME = "rmwebhook-remove";
+    private static final String LIST_WEBHOOK_EVENTNAME = "rmwebhook-list";
+    private static final String LINK_WEBHOOK_EVENTNAME = "rmwebhook-link";
+    private static final String UNLINK_WEBHOOK_EVENTNAME = "rmwebhook-unlink";
+
+    /**
+     * Factory to create ConnectReleaseWebHookEvent.
+     */
+    public static class Factory implements AbstractHookEvent.Factory {
+        @Override
+        public ConnectReleaseWebHookEvent create() {
+            return new ConnectReleaseWebHookEvent();
+        }
+
+        @Override
+        public String getSampleRequestPayload() {
```

Comment:
where is this used?

Reply:
this isn't used anywhere, but need to be overwritten.